### DISCUSSION TITLE
How do I hide old courses?

### DISCUSSION THREAD

**Discussion by doej1367** [ls1intum/Artemis#4191] [Q&A] [answered]
Is there an option to hide old courses that I have signed up / registered for in the last semester or some even years ago (like e.g. the tutorial course that has been there for years now)?

Im asking because it becomes quite cluttered over time and I would like to only see the two courses that are important right now - not the many I have signed up for over the past years.

I'm using the TUM version btw - just in case that makes a difference.

Thanks in advance

EDIT: even though old courses are automatically hidden as [mentioned here](https://github.com/ls1intum/Artemis/issues/38), it does not seem to work with some

**Comment by jpbernius** [comment, marked as answer]
The course instructors can define the time period in which a course is active. Once this period has passed, the course is hidden. If a course does not disappear after the semester, it is because the instructors have defined a future date as the end period, or did not set an end period at all. The `Course Information` box shows the start and end dates for the course.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/6382716/142600350-a66c26e7-a656-41de-8df5-61a242d3bb87.png">
